pkg/base1/cockpit.js

```diff
@@ -1606,9 +1630,7 @@ function basic_scope(cockpit, jquery) {
                 callback.call(self, row, x, n);
             }
 
-            var event = document.createEvent("CustomEvent");
-            event.initCustomEvent("notify", false, false, [ x, n ]);
-            self.dispatchEvent(event, x, n);
+            self.dispatchEvent("notify", x, n);
```

Comment:
We don't use the details argument so far. But I fixed this anyway so it works for the general case where someone does use the details argument.